app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import { CONTACT_VALIDATION_SCHEMA, ROLE_OPTIONS } from "./constants";
+
+const ContactForm = ({ onClose, setContacts, contact, isEdit = false }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = values => {
+    const contactDetails = isEdit
+      ? { contact, ...values }
+      : { ...values, id: new Date().getTime(), created_at: "Feb, 5, 2021" };
```

Comment:
I am hoping that you kept the `...values` as the last item in the first obj and first item in the second obj intentionally. Both have different meanings. Ignore this comment if you already knew why positioning of spread matters.

Please read: https://www.bigbinary.com/react-best-practices/how-to-use-destructuring#bb-generated-p-id-5

Reply:
I put intentionally in case of edit I need the previous values and need to replace with edited values. Some values like created_at not available in the values object.